methods_test.go

```diff
@@ -279,14 +296,14 @@ func (s *ConfigParserSuite) TestGetBool(c *gc.C) {
 	newParser := configparser.New()
 	newParser.AddSection("testing")
 
-	for _, value := range []string{"1", "yes", "true", "on"} {
+	for _, value := range []string{"1", "yes", "true", "on", "True", "ON"} {
```

Comment:
The original list of strings comes from the Python ConfigParser documentation.

Reply:
Python also checks it in case-insensitive manner.

> These string values are checked in a case-insensitive manner.